src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -771,22 +771,22 @@ Limitation: Video title disappears when clicked."</string>
     <string name="revanced_preference_screen_player_buttons_title">Player buttons</string>
     <string name="revanced_preference_screen_player_buttons_summary">Hide or show buttons in the video player.</string>
 
-    <string name="revanced_hide_player_autoplay_button_title">Hide autoplay button</string>
+    <string name="revanced_hide_player_autoplay_button_title">Hide Autoplay button</string>
     <string name="revanced_hide_player_autoplay_button_summary_on">Autoplay button is hidden.</string>
     <string name="revanced_hide_player_autoplay_button_summary_off">Autoplay button is shown.</string>
-    <string name="revanced_hide_player_captions_button_title">Hide captions button</string>
+    <string name="revanced_hide_player_captions_button_title">Hide Captions button</string>
     <string name="revanced_hide_player_captions_button_summary_on">Captions button is hidden.</string>
     <string name="revanced_hide_player_captions_button_summary_off">Captions button is shown.</string>
-    <string name="revanced_hide_player_cast_button_title">Hide cast button</string>
+    <string name="revanced_hide_player_cast_button_title">Hide Cast button</string>
```

Comment:
These are not capitalized in ReVanced

Reply:
In my opinion, since it is a proper noun, it is better to capitalize it